pkgs/development/guile-modules/guile-gcrypt/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+, guile
+, libgcrypt
+, autoreconfHook
+, pkgconfig
+, texinfo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-gcrypt";
+  version = "0.2.1";
+
+  src = fetchurl {
+    url = "https://notabug.org/cwebber/${pname}/archive/v${version}.tar.gz";
+    sha256 = "1qj1yw0kman984x584jjjxnjdhm0pwgp09iyn3b5rqajx7svpqcd";
+  };
+
+  postConfigure = ''
+    sed -i '/moddir\s*=/s%=.*%=''${out}/share/guile/site%' Makefile;
+    sed -i '/godir\s*=/s%=.*%=''${out}/share/guile/ccache%' Makefile;
+  '';
+
+  nativeBuildInputs = [
+    autoreconfHook pkgconfig texinfo
+  ];
+  buildInputs = [
+    guile
+  ];
+  propagatedBuildInputs = [
+    libgcrypt
```

Comment:
Is that always required when using guile-gcrypt anywhere and would there need to be listed under buildInputs?

Reply:
I believe it is required.